Resources/Prototypes/Diseases/infectious.yml

```diff
@@ -48,6 +48,51 @@
     - !type:DiseaseReagentCure
       reagent: SpaceCleaner
 
+- type: disease
+  id: SpaceFlu
+  name: space flu
+  cureResist: 0.08
+  effects:
+    - !type:DiseaseVomit
+      probability: 0.01
+    - !type:DiseasePopUp
+      probability: 0.025
+    - !type:DiseaseSnough
+      probability: 0.025
+      snoughSound:
+        collection: Sneezes
+    - !type:DiseaseHealthChange
+      probability: 0.015
+      damage:
+        types:
+          Heat: 1
+  cures:
+    - !type:DiseaseBedrestCure
+      maxLength: 100
+
+- type: disease
+  id: Bird Flew
+  name: bird flew
+  cureResist: 0.08
+  effects:
+    - !type:DiseaseVomit
+      probability: 0.015
+    - !type:DiseasePopUp
+      probability: 0.025
+    - !type:DiseaseSnough
+      probability: 0.025
+      snoughMessage: disease-cough
+      snoughSound:
+        collection: Coughs
+    - !type:DiseaseHealthChange
+      probability: 0.05
+      damage:
+        groups:
+          Caustic: 1
+  cures:
+    - !type:DiseaseBedrestCure
+      maxLength: 120
```

Comment:
this seems like basically the same thing as space flu, does it need to exist

Reply:
Having similar diseases helps to own the metagamers and require you to do at least a bit of differentiation

Reply:
sure why not